inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sink/StreamSinkServiceImpl.java

```diff
@@ -254,15 +238,6 @@ public Boolean update(SinkRequest request, String operator) {
         StreamSinkOperator sinkOperator = operatorFactory.getInstance(request.getSinkType());
         sinkOperator.updateOpt(request, nextStatus, operator);
 
-        // If the stream is [CONFIG_SUCCESSFUL], then asynchronously start the [CREATE_STREAM_RESOURCE] process
```

Comment:
So what's the appropriate time to restart after config updates ? Perhaps it's better be launched by end user in a separate call. According to the Single Responsibility Principle the service update handler should only perform no other than config updating.